src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -189,6 +191,7 @@ class AddressValidationModal extends React.Component {
         id="address-validation-warning"
         onClose={closeModal}
         visible={isAddressValidationModalVisible}
+        focusSelector="usa-button-primary"
```

Comment:
Is this gonna make other elements on the modal unfocusable? ie will it override the defaults? https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/modal/

Reply:
No - its just the initial focus

Reply:
Cool. That sounds familiar. Is this consistent with how other modals work? Might want to run it past Arthur